consul/docker-entrypoint.sh

```diff
@@ -10,11 +10,16 @@ haproxytemplate="/consul/template/haproxy/haproxy.cfg.ctmpl:/consul/template/hap
 /bin/consul-template       -template=$caddytemplate -template=$haproxytemplate &
 
 # adapt the docker group of the container to the outside
-DOCKER_GID=$(stat -c %g /var/run/docker.sock)
+DOCKER_GID=$(stat -c %g /run/docker.sock)
```

Comment:
Why change this path ? As long I can see on last version of coreos stable we have:

```bash
core@core4 ~/cluster $ systemctl cat docker.socket
# /run/systemd/system/docker.socket
[Unit]
Requires=torcx.target
After=torcx.target
Description=Docker Socket for the API
PartOf=docker.service

[Socket]
ListenStream=/var/run/docker.sock
SocketMode=0660
SocketUser=root
SocketGroup=docker

[Install]
WantedBy=sockets.target
```

Reply:
Ok, right ``/var/run`` is a symlink to ``/run``